class-00/lab-b/starter-code/app.js

```diff
@@ -231,7 +231,9 @@ console.log('Joe scope arrow:', joe.scopeArrow());
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
 // It is undefined.

```

Comment:
The 'this' inside of joe.scope method is not undefined. In your console you will see a log from line 216 reporting it. You will also see a log from line 222 which is logging of the return value of the joe.scope() method, and this is undefined.